lib/runnerman.js

```diff
@@ -21,77 +22,85 @@ module.exports = async function (suite, collectionDefinition, environment, itera
+                            const newObj = Object.create();
                             newObj[key] = value(err, summary, runner.summary);
-
-                            Object.assign(data, newObj);
+                            Object.assign(result, newObj);
                         }
                     });
                 }
 
-                //TODO: Remover this variables in "item"
-                Object.assign(runner.summary, createGlobalVariable(data, summary.cursor.position, runner.summary));
+                // TODO: Remover this variables in "item"
+                Object.assign(runner.summary,
+                    createGlobalVariable(result, summary.cursor.position, runner.summary));
             }
         });
 
-        runner.on('request', function (err, summary) {
-            const response = summary.response;
+        runner.on('request', (err, summary) => {
+            const { response } = summary;
             if (response !== undefined) {
                 const buffer = response.stream;
 
                 const variable = globals.values.members[summary.cursor.position].value;
 
+                const result = summary.response.code === variable.code;
+
                 const event = new Event({
                     listen: 'test',
                     script: new Script({
-                        exec: format(
-                            'tests["%s"] = %s',
+                        exec: format('tests["%s"] = %s',
                             guid,
-                            new Boolean(summary.response.code === variable.code).toString()
-                        )
+                            result.toString())
                     })
                 });
 
                 summary.item.events.members.push(event);
 
                 if (summary.response.code === variable.code && buffer.byteLength > 0) {
-                    Object.assign(
-                        runner.summary,
-                        createGlobalVariable(JSON.parse(buffer.toString()), summary.cursor.position, runner.summary)
-                    );
+                    Object.assign(runner.summary,
+                        createGlobalVariable(JSON.parse(buffer.toString()), summary.cursor.position, runner.summary));
                 }
             }
         });
 
-        runner.on('exception', function (err) {
+        runner.on('assertion', err => {
             if (err) {
+                consoleColor = '\x1b[31m';
+            }
+        });
+
+        runner.on('exception', err => {
+            if (err) {
+                consoleColor = '\x1b[31m';
                 reject(runner.summary);
                 throw err;
             }
         });
 
-        runner.on('done', function (err, summary) {
-            console.log(format('Suite %s run complete!', name));
+        runner.on('done', () => {
+            console.log(`\x1b[0m\x1b[1m${consoleColor}`, `'Suite ${name} run complete!'`);
+
             resolve(runner.summary);
```

Comment:
return quando mudar para async func

Reply:
See this [comment](https://github.com/FCesar/runnerman/pull/4#discussion_r432911614).